pyiron_contrib/RDM/storagejob.py

```diff
@@ -0,0 +1,337 @@
+        pass
+
+    def run_if_refresh(self):
+        pass
+
+    def _run_if_busy(self):
+        pass
+
+    def write_input(self):
+        pass
+
+    def reconnect_s3_storage(self, config=None, bucket_name=None):
+        config = config or self._storage_type.s3_config.config
+        bucket_name = bucket_name or self._storage_type.s3_config.bucket_name
+        try:
+            external_storage = FileS3IO(
+                config=config,
+                path=self.path,
+                bucket_name=bucket_name
+            )
+        except Exception as e:
+            raise RuntimeError("Could not restore connection to the S3 storage.") from e
+        if self._storage_type.s3_config.bucket_info == external_storage.bucket_info:
+            self._external_storage = external_storage
+        else:
+            raise RuntimeError(f"New and saved s3 storage do not match! Got {external_storage.bucket_info}"
+                               f" but expected {self._storage_type.s3_config.bucket_info}.")
+
+    def from_hdf(self, hdf=None, group_name=None):
+        super().from_hdf(hdf, group_name)
+        self._stored_files.from_hdf(hdf=self._hdf5)
+        self._storage_type.from_hdf(hdf=self._hdf5)
+        if self._storage_type.s3 and self.status != "initialized":
+            self.reconnect_s3_storage()
+
+    def to_hdf(self, hdf=None, group_name=None):
+        super().to_hdf(hdf, group_name)
+        if self._storage_type.s3:
+            self._hdf5['REQUIRE_FULL_OBJ_FOR_RM'] = True
+        self._stored_files.to_hdf(hdf=self._hdf5)
+        self._storage_type.to_hdf(hdf=self._hdf5)
+
+    def __getitem__(self, item):
+        # copied from super().__getitem__ changing the output of returning a file
+        """
+        Get/read data from the HDF5 file, child jobs or access log files.
+
+        If the job is :method:`~.decompress`ed, item can also be a file name to
+        access the raw output file of that name of the job.  See available file
+        with :method:`~.list_files()`.
+
+        `item` is first looked up in this jobs HDF5 file, then in the HDF5 files of any child jobs and finally it is
+        matched against any files in the job directory as described above.
+
+        Args:
+            item (str, slice): path to the data or key of the data object
+
+        Returns:
+            dict, list, float, int, None: data or data object; if nothing is found None is returned
+        """
+        try:
+            return self._hdf5[item]
+        except ValueError:
+            pass
+
+        name_lst = item.split("/")
+        item_obj = name_lst[0]
+        if item_obj in self._list_ext_childs():
+            child = self._hdf5[self._name + "_hdf5/" + item_obj]
+            print("job get: ", self._name + "_jobs")
+            if len(name_lst) == 1:
+                return child
+            else:
+                return child["/".join(name_lst[1:])]
```

Comment:
This 'job' will never have children, so I'd think this can go.
```suggestion
```

Reply:
Actually, it will also never have sensible (i.e. user-facing) data in HDF either, right?

Reply:
So far not planned. Could just provide a quick access to the meta data only.